### PR TITLE
Update powershell connector release notes

### PR DESCRIPTION
**Pull request by alexandruatomei9**

### CODE REVIEW

File: modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc [hunk 1]
```diff
@@ -9,6 +9,25 @@ _Select_
 
 PowerShell is a Windows command line shell for system administrators. Windows PowerShell includes an interactive prompt and a scripting environment that can be used independently or in combination.
```
Comment (dhirsch1) [inline modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc]: Please change the title to this: 

PowerShell Connector Release Notes - Mule 4

Comment (dhirsch1) [inline modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc]: In combination with what?

File: modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc [hunk 2]
```diff
@@ -9,6 +9,25 @@ _Select_
...
```
Comment (dhirsch1) [inline modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc]: One more fix to the existing text. Replace _Select_ with this text:

Support Category: Select (with no italics)

File: modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc [hunk 3]
```diff
@@ -9,6 +9,25 @@ _Select_
...
+[%header%autowidth.spread]
+|===
+|Application/Service |Version
+|Mule Runtime |4.0.0 and later
+|Java |1.8 and later
+|===
+
+=== Fixed in this release
+
+* Powershell error stop logging anything on the console.(SE-11546)
```
Comment (dhirsch1) [inline modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc]: Please use this text: Powershell stopped logging messages on the console. (SE-11546)

Comment (alexandruatomei9) [reply modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc]: This is how the ticket was opened. Should we really rephrase it?

File: modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc (hunk 2 above)
Comment (alexandruatomei9) [reply modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc]: done

File: modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc (hunk 1 above)
Comment (alexandruatomei9) [reply modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc]: done

Comment (alexandruatomei9) [reply modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc]: done

File: modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc (hunk 3 above)
Comment (dhirsch1) [reply modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc]: We should keep it the same, but it is ungrammatical, so change "stop" to "stopped"

Comment (dhirsch1) [reply modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc]: Also, there should be a space before (SE-11546).

Comment (alexandruatomei9) [reply modules/ROOT/pages/connector/microsoft-powershell-connector-release-notes-mule-4.adoc]: done

**Pull request merged**